crypto/spock_test.go

```diff
@@ -154,4 +163,17 @@ func TestSPOCKProveVerify(t *testing.T) {
 		assert.True(t, IsNotBLSKeyError(err))
 		assert.False(t, result)
 	})
+
+	// test with identity public key and proof
+	t.Run("identity proof", func(t *testing.T) {
+		identityProof := make([]byte, signatureLengthBLSBLS12381)
+		identityProof[0] = identityBLSSignatureHeader
+		result, err := SPOCKVerify(IdentityBLSPublicKey(), identityProof, sk2.PublicKey(), pr2)
+		assert.NoError(t, err)
+		assert.False(t, result)
+
+		result, err = SPOCKVerify(sk1.PublicKey(), pr1, IdentityBLSPublicKey(), identityProof)
+		assert.NoError(t, err)
+		assert.False(t, result)
```

Comment:
Suggesting to test the case whether both pairs are identity key/proof
```suggestion
		assert.False(t, result)
		
		result, err = SPOCKVerify(IdentityBLSPublicKey(), identityProof, IdentityBLSPublicKey(), identityProof)
		assert.NoError(t, err)
		assert.False(t, result)
```

Reply:
Yep, good suggestion!